reporting/api/src/main/java/org/jboss/windup/reporting/model/JarDependenciesReportModel.java

```diff
@@ -0,0 +1,33 @@
+package org.jboss.windup.reporting.model;
+
+import org.jboss.windup.graph.model.ProjectModel;
+
+import com.tinkerpop.blueprints.Direction;
+import com.tinkerpop.frames.Incidence;
+import com.tinkerpop.frames.modules.typedgraph.TypeValue;
+
+/**
+ * This represents Jar Dependencies report model
+ * 
+ * @author mnovotny
+ *
+ */
+@TypeValue(JarDependenciesReportModel.TYPE)
+public interface JarDependenciesReportModel extends ApplicationReportModel
+{
+    String TYPE = "jarDependenciesReport";
+
+    String JAR_REPORT_TO_PROJECT_MODEL = "jarDepsToProjectModel"; 
+
+    /**
+     * Contains all {@link ProjectModel}s that contain this file
+     */
+    @Incidence(label = JAR_REPORT_TO_PROJECT_MODEL, direction = Direction.OUT)
```

Comment:
This seems to create a link directly from the report to a JAR. I'm not sure that this will cover the relationship adequately as it seems that a report will have multiple groups of jars. It seems another vertex in between the report and the actual JAR might be useful.

If the words above are confusing, a picture might be clearer: https://docs.google.com/drawings/d/1mxGZN210tUEqie9fdqOMb41O3jz6HDyI8ZLNyhkTLJQ/edit